image_proc/launch/image_proc.launch.py

```diff
@@ -41,27 +41,35 @@
 
 
 def generate_launch_description():
+
+    arg_namespace = DeclareLaunchArgument(
+        name='namespace', default_value='',
+        description=('namespace for all components loaded')
+    )
+
     composable_nodes = [
         ComposableNode(
             package='image_proc',
             plugin='image_proc::DebayerNode',
             name='debayer_node',
+            namespace=LaunchConfiguration('namespace'),
         ),
         ComposableNode(
             package='image_proc',
             plugin='image_proc::RectifyNode',
             name='rectify_mono_node',
+            namespace=LaunchConfiguration('namespace'),
             # Remap subscribers and publishers
             remappings=[
                 ('image', 'image_mono'),
-                ('camera_info', 'camera_info'),
```

Comment:
we should not remove this - while it isn't doing much in this particular example, Iron does not have the fixes for properly remapping camera_info topics alongside the image topic (and it won't get those fixes, since it would possibly break anyone who has manually remapped them appropriately)